src/js/stores/MesosStore.js

```diff
@@ -0,0 +1,188 @@
+import {EventEmitter} from "events";
+
+import AppDispatcher from "../AppDispatcher";
+import InfoStore from "../stores/InfoStore";
+import InfoActions from "../actions/InfoActions";
+import InfoEvents from "../events/InfoEvents";
+import MesosActions from "../actions/MesosActions";
+import MesosEvents from "../events/MesosEvents";
+
+const FILES_TTL = 60000;
+const STATE_TTL = 60000;
+const MASTER_ID = "master";
+
+class MesosStore extends EventEmitter {
+
+  constructor() {
+    super();
+    this._state = {};
```

Comment:
The Store is a singleton, so it doesn't make sense to bind this vars at the object-instances.
Please put them in the private scope outside the MesosStore-object.